File: app/src/main/java/com/zhong/cardinals/sample/login/LoginView.java
package com.zhong.cardinals.sample.login;

import com.zhong.cardinals.mvp.MvpView;
import com.zhong.cardinals.sample.mode.UserInfo;

/**
 * Created by Mr.zhong on 2017/9/19.
 */

public interface LoginView extends MvpView {
    void isEmptyPhoneNumber();

    void isEmptyPassword();

    void showDialog();

    void closeDialog();

    void onSuccess(UserInfo userInfo);

    void onFaile(int code, String msg);
}

File: app/src/main/java/com/zhong/cardinals/sample/login/LoginView.kt
package com.zhong.cardinals.sample.login

import com.zhong.cardinals.mvp.MvpView
import com.zhong.cardinals.sample.mode.UserInfo

/**
 * Created by Mr.zhong on 2017/9/19.
 */

interface LoginView : MvpView {
    fun isEmptyPhoneNumber()

    fun isEmptyPassword()

    fun showDialog()

    fun closeDialog()

    fun onSuccess(userInfo: UserInfo?)

    fun onFaile(code: Int, msg: String)
}
